docs/config/aws-configuration.rst

```diff
@@ -20,6 +20,20 @@ The configuration item to use AWS should look like:
 Make sure that the ``region`` and ``zone`` match each other and that both match the region and zone where the Flocker agent nodes run.
 AWS must be able to attach volumes created in that availability zone to your Flocker nodes.
 
+In addition to the mandatory properties shown, there are some optional properties:
+
+.. option:: session_token
+
+   An AWS session token.
+   This allows cross-account access.
+   It is mainly useful for testing since session tokens only last for a short time.
```

Comment:
Is it possible to say how you will know you will need this, or where to get one if you do?